wiki/Beatmap/id.md

```diff
@@ -1,12 +1,118 @@
 ---
-outdated: true
-outdated_since: f4d914b0b826e19e50caeea90f58557065e9e0b7
+tags:
+  - mapset
+  - beatmapset
 ---
 
 # Beatmap
 
-*Artikel ini tidak mencakup seputar [Tingkat Kesulitan Beatmap](/wiki/Beatmap/Difficulty) secara mendalam.*
+**Beatmap** (terkadang disebut sebagai *beatmapset*) adalah sebuah set yang memiliki level gim (atau [tingkat kesulitan](#difficulty)) yang terdiri dari berbagai [hit object](/wiki/Hit_object) yang hampir selalu mewakili suatu lagu. Selain mewakili suatu lagu, beatmap juga mencakup komponen lainnya, semua dikemas dalam bentuk arsip [berekstensi .osz](/wiki/osu!_File_Formats/Osz_(file_format)):
 
-**Beatmap** merupakan istilah yang merujuk pada level-level permainan yang tersedia untuk dimainkan. Tiap-tiap berkas beatmap mengandung berbagai informasi yang terkait dengan aspek-aspek yang hadir di dalam permainan seperti [*hit object*](/wiki/Hit_Objects), [*timing*](/wiki/Beatmap_Editor/Timing), metadata, pengaturan tingkat kesulitan *(difficulty setting)*, efek-efek visual, dan lain sebagainya.
+- berkas lagu itu sendiri, yang tersimpan dalam format MP3 atau Ogg.
+- [background/gambar latar belakang](wiki/Beatmap/Background), atau sebuah video, yang berperan sebagai playfield.
+- [custom hitsound](/wiki/Beatmapping/Hitsound) yang digunakan untuk aransemen dan umpan balik aural yang ditingkatkan (opsional).
+- [storyboard](/wiki/Storyboards) dengan efek-efek visual, berfungsi sebagai latar cerita atau tema lagu (opsional).
+- [custom skin](/wiki/Skinning), yang mengubah tampilan elemen antarmuka dan gameplay (opsional).
 
-Beatmap-beatmap yang dibuat untuk satu berkas lagu yang sama pada umumnya akan secara otomatis terkelompokkan ke dalam satu **[beatmapset](Beatmapsets)** utuh. Kamu dapat menjumpai beragam jenis beatmapset yang telah [diunggah ke situs osu!](/wiki/Glossary#bss) pada laman [daftar beatmap](https://osu.ppy.sh/beatmapsets), di mana pada masing-masing beatmapset tersebut kamu akan dihadapkan dengan satu atau lebih [tingkat kesulitan](/wiki/Beatmap/Difficulty) di dalamnya.
+*Catatan: beberapa fitur visual dan aural pada sebuah beatmap dapat dinonaktifkan melalui [visual settings](/wiki/Visual_Settings).*
```

Comment:
```suggestion
*Catatan: Beberapa fitur visual dan aural pada sebuah beatmap dapat dinonaktifkan melalui [visual settings](/wiki/Visual_Settings).*
```

Reply:
applied for this, but the rest are not. they're written in lowercase as we can see the en.md.